app/views/api/_image.xml.builder

```diff
@@ -13,16 +13,17 @@ xml.tag!(tag,
   xml_datetime(xml, :last_viewed, object.last_view)
   xml_boolean(xml, :ok_for_export, true) if object.ok_for_export
   xml_string(xml, :original_name, object.original_name) if check_permission(object)
-  xml_detailed_object(xml, :license, object.license)
+  xml.observations(number: object.observations.length) do
+    for id in object.observation_ids
```

Comment:
You would offend RuboCop 😭  if RuboCop had looked at this page. 😁 
RuboCop would like:
```suggestion
    object.observation_ids.each do |id|
```
I wonder if I could get RuboCop to look at .xml.builder pages. (Not likely.)